Kickstarter-iOS/Views/Controllers/RewardPledgeNavigationController.swift

```diff
@@ -32,9 +31,21 @@ extension RewardPledgeNavigationController: UINavigationControllerDelegate {
     case (.pop, is PledgeViewController, is RewardPledgeTransitionAnimatorDelegate):
       return RewardPledgePopTransitionAnimator()
     default:
-      // We're not performing any custom transition, bring back the standard divider line
```

Comment:
This was only necessary for showing the non-native pledge view, if we still want to do this it should be via feature flag.